README.md

```diff
@@ -1,2 +1,70 @@
 # My_Portfolio
 Setup and mobile first
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name  
+
+> This is my Portfolio project
+
+
+## Built With
+
+- HTML and CSS
+- I used style linter
+ 
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [Optional] Consider adding the steps required to get a local copy of your project in this section. You can remove sections that do not apply to your work.